FreeRTOS/Demo/CORTEX_R4_RM48_TMS570_CCS5/CreateProjectDirectoryStructure.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
```

Comment:
Should we instead remove both these scripts and  use linked resources instead?

Reply:
Yeah I would second this. Just symlink the resources that are copied and get rid of the scripts. The only real caveat against this is that prior to Vista, Windows did not support symlinks, but I would consider that to be a negligible issue.

Reply:
Sounds good. I will update this PR with that change instead. I have it working locally after a bit of messing around with relative paths.

Where does the documentation live? [This section in the docs](https://freertos.org/Documentation/02-Kernel/03-Supported-devices/04-Demos/Texas-Instruments/Free_RTOS_for_TI_RM48_and_TMS570#preparing-the-code-composer-studio-eclipse-project-directory) will also need updating since this script will no longer exist.

Reply:
Hi @mryall-mawson,
I will update the documentation on the website for you.
Best,
Jason Carroll